.github/workflows/update-changelog.yml

```
@@ -1,51 +1,54 @@
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: Get version from tag
-        id: get-version
-        uses: battila7/get-version-action@v2
+          ref: ${{ github.head_ref }}
+
+      - name: Get version info
+        id: version_info
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:38: Double quote to prevent globbing and word splitting

(shellcheck)